lms/tasks/email_digests.py

```diff
@@ -50,21 +50,19 @@ def send_instructor_email_digest_tasks(*, batch_size):
                 .join(ApplicationInstance)
                 .join(AssignmentMembership)
                 .join(LTIRole)
-                .outerjoin(
-                    EmailUnsubscribe,
-                    and_(
-                        User.h_userid == EmailUnsubscribe.h_userid,
-                        EmailUnsubscribe.tag == EmailUnsubscribe.Tag.INSTRUCTOR_DIGEST,
-                    ),
-                )
                 .where(
                     ApplicationInstance.settings["hypothesis"][
                         "instructor_email_digests_enabled"
                     ].astext
                     == "true",
                     LTIRole.type == "instructor",
                     # No EmailUnsubscribes
-                    EmailUnsubscribe.id.is_(None),
+                    User.h_userid.not_in(
```

Comment:
This is tested over:

https://github.com/hypothesis/lms/blob/1f99780d739112383404cc820df5fc5358c3f546/tests/unit/lms/tasks/email_digests_test.py#L61